README.md

```diff
@@ -37,10 +37,26 @@ For more detailed usage, see [the guide](https://github.com/brendanarnold/py-for
 - At present the library mimics the IO of the Intel FORTRAN compiler
   v.9.1 run on a Linux system. Differences to other FORTRAN compilers
   and platforms are generally minor.
-- The library should run on Python versions from at least 3.6
+- `pytest` requires at least Python 3.8.
 
 ## Development
 
+### Installing locally
+
+First, make sure that you have installed `poetry`. You can find the build instructions at [python-poetry.org/docs](https://python-poetry.org/docs/#installing-with-the-official-installer). For most users on Linux/UNIX, you can run the following in a terminal
+```
+curl -sSL https://install.python-poetry.org | python3 -
+```
+Then, start a new terminal window and try run `poetry --version` to make sure that `poetry` has been linked correctly. 
+
+```
+cd path/to/fortranformat
+# Install the project
+poetry install
+# Run pytest and make sure these pass
+poetry run pytest
```

Comment:
Adding some spacing and syntax highlighting to the shell

```suggestion
```bash
cd path/to/fortranformat

# Install the project
poetry install

# Run pytest and make sure these pass
poetry run pytest
```